### PR TITLE
Upgrade of samples to MonoGame 3.8

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
All samples have been upgraded to MonoGame 3.8

* Platformer 2D upgraded - adheres to new MG template standard
* New documentation and tutorial from the original XNA tutorial for the Platformer sample
* NeonShooter upgraded - adheres to new MG template standard

* SpaceWar sample removed, too many issues

### CODE REVIEW

File: NeonShooter/NeonShooter.Android/Content.mgcb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#----------------------------- Global Properties ----------------------------#
+
+/outputDir:bin/$(Platform)
+/intermediateDir:obj/$(Platform)
+/platform:iOS
```
Comment (dellis1972) [inline NeonShooter/NeonShooter.Android/Content.mgcb]: should this be Android?

**Comment by SimonDarksideJ** [comment]
That shouldn't be there more likely, as the content project is in the shared lib :D,  
Great spotting @dellis1972 , will double-check and clean up.

File: NeonShooter/NeonShooter.Android/Content.mgcb (hunk 1 above)
Comment (SimonDarksideJ) [reply NeonShooter/NeonShooter.Android/Content.mgcb]: Removed all not required default content

**Comment by CartBlanche** [comment]
Nice!

**Pull request merged**